homeassistant/components/plex/models.py

```diff
@@ -70,7 +70,7 @@ def update_media(self, media):
             self.media_library_title = "Live TV"
         else:
             self.media_library_title = (
-                media.section().title if media.section() is not None else ""
+                media.section().title if media.librarySectionID is not None else ""
```

Comment:
Don't you need to keep the `None` check for `media.section()`?

Reply:
Unfortunately calling `section()` when the media's `librarySectionID` is `None` throws an exception in the library:
https://github.com/pkkid/python-plexapi/blob/1149c7f2488660e9e27d98a66a841812574480b2/plexapi/base.py#L494-L496
and
https://github.com/pkkid/python-plexapi/blob/c9b2c3628eed9350dd0f55bfa459393d5ffdd518/plexapi/library.py#L61-L69